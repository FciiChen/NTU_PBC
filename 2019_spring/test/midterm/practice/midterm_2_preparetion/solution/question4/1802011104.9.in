不是在說那個營運不到一年的BRT

而是在說放送至今一年半多的

從惡作劇博取目光的屁癲小子

到交朋友找樂子毫無目的混日子

除了有義氣看重同儕外的8+9頭子

看不到有什麼足以說是吸引人的人格特質


而現在巳月叛逃至大野木麾下的當下

BRT也終於顯現出就算用暴力也要把人帶回來的決心

就像當初NRT組隊抓SSK那樣

不講理我想幹嘛就幹嘛就是我的忍道

相信我就對了之術作為了自己的人生價值


走上跟先輩同樣的套路後

BRT有機會大復活成為眾人焦點嘛？
BREAK