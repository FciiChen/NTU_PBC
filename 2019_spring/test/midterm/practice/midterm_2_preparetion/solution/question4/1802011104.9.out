B_T 3
幹_就 2
是_說 2
而_在 2
9_子 1
K_樣 1
N_T 1
R_也 1
R_有 1
R_組 1
